incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+        }
+
+        code {
+            font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+            font-size: 87.5%;
+            color: #e83e8c;
+            word-break: break-word;
+        }
+
+        .left-column {
+            padding: .75rem;
+            max-width: 75%;
+            min-width: 55%;
+        }
+
+        .right-column {
+            padding: .75rem;
+            max-width: 25%;
+        }
+
+        .container {
+            display: flex;
+            width: 100%;
+        }
+
+        li {
+            margin: 0.75rem;
+        }
+
+        .right-section {
+            margin-left: 1rem;
+            padding-left: 0.5rem;
+        }
+
+        .right-section h3 {
+            padding-top: 0;
+            font-weight: 200;
+        }
+
+        .right-section ul {
+            border-left: 0.3rem solid #00A1E2;
+            list-style-type: none;
+            padding-left: 0;
+        }
+
+    </style>
+</head>
+<body>
+
+<div class="banner lead">
+    Your new Cloud-Native application is ready!
+</div>
+
+<div class="container">
+    <div class="left-column">
+        <p class="lead"> Congratulations, you have created a new Quarkus application.</p>
+
+        <h2>Why do you see this?</h2>
+
+        <p>This page is served by Quarkus. The source is in
+            <code>src/main/resources/META-INF/resources/index.html</code>.</p>
+
+        <h2>What can I do from here?</h2>
+
+        <p>If not already done, run the application in <em>dev mode</em> using: <code>mvn quarkus:dev</code>.
+        </p>
+        <ul>
+          <li>Add REST resources, Servlets, functions and other services in <code>src/main/java</code>.</li>
+          <li>Your static assets are located in <code>src/main/resources/META-INF/resources</code>.</li>
+          <li>Configure your application in <code>src/main/resources/application.properties</code>.
+          </li>
+        </ul>
+
+        <h2>How do I get rid of this page?</h2>
+        <p>Just delete the <code>src/main/resources/META-INF/resources/index.html</code> file.</p>
+    </div>
+    <div class="right-column">
+        <div class="right-section">
+            <h3>Application</h3>
+            <ul>
+                <li>GroupId: org.acme.quarkus.sample</li>
+                <li>ArtifactId: kafka-quickstart</li>
+                <li>Version: 1.0-SNAPSHOT</li>
+                <li>Quarkus Version: 1.6.0.Final</li>
+            </ul>
+        </div>
+        <div class="right-section">
+            <h3>Next steps</h3>
+            <ul>
+                <li><a href="https://quarkus.io/guides/maven-tooling.html">Setup your IDE</a></li>
+                <li><a href="https://quarkus.io/guides/getting-started.html">Getting started</a></li>
+                <li><a href="https://quarkus.io">Quarkus Web Site</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+
+</body>
+</html>
```

Comment:
Can you add the new-line to the end of the file? We use diff to make sure the code templates stay in sync with the Quarkus quickstarts they are based on. This flags the file as having differences, so will make maintenance harder when we update the Quarkus version and the template code. 

I fixed this on my Mac with:

```
sed -i '' -e '$a\' incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html
```

I'm not smart enough to know why adding a new line in vi didn't work for me!

Reply:
I have added this now :) thanks!